chapters/packages.tex

```diff
@@ -228,9 +228,9 @@ \subsection{Directory Hierarchy Mapping}\label{mapping-a-package-class-hierarchy
 \subsection{Single File Mapping}\label{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}\label{single-file-mapping}
 
 When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), that file shall only define a single class \lstinline!A! with a name matching the name of the nonstructured entity.
-In a file hierarchy the files shall have the extension \filename{.mo}.
+In a file hierarchy the files shall have the extension \filename{mo}.
 
-A \filename{.mo} file defining more than one class cannot be part of the mapping
+A \filename{mo} file defining more than one class cannot be part of the mapping
```

Comment:
This is a situation where I would not have identified this as a filename extension, but as an often used informal pattern to refer to files with a given extension, and then I would have included the dot.

Reply:
Removed filename in this case.